tests/common/plugins/conditional_mark/issue.py

```diff
@@ -87,6 +109,27 @@ def is_active(self):
         return True
 
 
+def is_file_empty_or_comments(file_path):
```

Comment:
If use `yaml.safe_load` to load a yaml file with only comments, will it raise exception?

Reply:
it won't, it will return None